test/expect-tests/dune_console/dune_console_tests.ml

```diff
@@ -74,19 +135,27 @@ let%expect_test "basic usage" =
   let module Console = New () in
   Console.Backend.set Console.Backend.progress;
   test_basic_usage (module Console);
+  escape [%expect.output];
   [%expect
     {|
-    Hello World!
-    Hello this is a very long sentence that will probably wrap the console by the
-    time that this is over. |}]
+Hello World!
+Hello this is a very long sentence that will probably wrap the console by the
+time that this is over.
+  |}]
 ;;
 
 let%expect_test "Status line clearing." =
   let module Console = New () in
   Console.Backend.set Console.Backend.progress;
   test_status_line_clearing (module Console);
+  escape [%expect.output];
   [%expect
-    "Here is a status line\r                     \rHere is another\r               \r"]
+    {| 
+Here is a status line\r
+                     \r
+Here is another\r
+               \r
+ |}]
```

Comment:
The extra newline after the return shows that we are going back the correct amount. We need two returns when updating the status. The first so that we can clear the cells and the second after we are done.

Reply:
Just use `String.escaped`. The current test is barely readable to me. I'd rather look at escape codes than count spacing.

Reply:
Since I am going to get rid of the spacing in the fix, can we keep it this way for the time being? The reason I am asking for it is because it really isn't easy to see if the spacing is correct when it is not on a new line.

Reply:
Also, I am not putting the spacing in myself, this is done by the console. The escape chars are exactly `\r`.

Reply:
Okay, then push your fix to the current branch then. You're saying it isn't to see, well maybe you should compute the property you're trying to test instead of making the reader eyeball it.

Reply:
It won't make sense to compute the number of spaces later since we won't be clearing lines that way.